11-query/03-insert-query.md

```diff
@@ -80,6 +80,55 @@ This `match insert` query:
 3. Inserts an `employment` relation with `$org` and `$person` as its roleplayers, assigned to variable `$new-employment`.
 4. Inserts the ownership of `reference-id` with value `WGFTSH` to the `$new-employment` relation instance.
 
+### Duplicate Role Players
+
+As of Grakn 1.7.0, relations are allowed to have duplicate role players. This means the following would be valid in Grakn:
+
+<div class="tabs dark">
+
+[tab:Graql]
+```graql
+match
+  $org isa organisation, has name "Facelook";
+  $person isa person, has email "tanya.arnold@gmail.com";
+insert $fake-duplicate-employment (employer: $org, employee: $person, employee: $person) isa employment;
+  $new-employment has reference-id "WGFTSH";
```

Comment:
do you think this query is helpful in understanding the use case of inserting multiple roleplayers?

Reply:
i think friendship might be slightly better as an example, is this code tested using `social_network`?

Reply:
> i think friendship might be slightly better as an example

I agree

> is this code tested using `social_network`?

yes

Reply:
cool, thanks